pkgs/os-specific/linux/kernel/patches.nix

```diff
@@ -62,4 +62,10 @@
     name = "fix-em-ice-bonding";
     patch = ./fix-em-ice-bonding.patch;
   };
+
+  # https://lore.kernel.org/linux-wireless/ZAx0TWRBlGfv7pNl@kroah.com/T/#m11e6e0915ab8fa19ce8bc9695ab288c0fe018edf
+  fix-brcmfmac = {
+    name = "fix-brcmfmac";
+    patch = ./fix-brcmfmac.patch;
```

Comment:
Could we fetch from https://lore.kernel.org/linux-wireless/20230311141914.24444-1-marcan@marcan.st/raw rather than vendoring?